main.tf

```diff
@@ -60,6 +60,33 @@ locals {
 
   subnet_name_pod_cidr     = "gitlab-cluster-pod-cidr"
   subnet_name_service_cidr = "gitlab-cluster-service-cidr"
+
+  default_pool = [{
+    name                       = "gitlab"
```

Comment:
should we just call it default pool ?

Reply:
changed to gke_gitlab_node_pool

Reply:
If I understood well, applying this will replace the current node pool. Right ?